src/main/telemetry/ibus.c

```diff
@@ -277,12 +282,13 @@ static void dispatchMeasurementRequest(ibusAddress_t address) {
 
     case IBUS_SENSOR_TYPE_TEMPERATURE:
         {
+            //TODO: handle arithmetric better
             #ifdef BARO
-                float temperature = (baroTemperature + 50) / 100.f;
+                uint16_t temperature = (baroTemperature + 50) / 10;
```

Comment:
Frsky code rounds the value, it works fine for non-negative temperatures. But added value has to be half the divisor.  
`baroTemperature / 10` should do in iBus case.